.gitlab-ci.yml

```diff
@@ -309,7 +309,7 @@ deploy-feature:
   stage: deploy
   when: manual
   variables:
-    SUFFIX: 'my-feature' #/datadog-[product]-${SUFFIX}.js
+    SUFFIX: 'lazy-replay' #/datadog-[product]-${SUFFIX}.js
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏ You can set the suffix in gitlab when you start a manual job

Reply:
Nice, good to know :)